e2e/cypress/index.d.ts

```diff
@@ -7,9 +7,8 @@ declare namespace Cypress {
     saveStorage(): void
     restoreStorage(): void
     applyProjectsFilter(projectsToFilterOn: string[]): void
-    generateAdminToken(idToken: string): void
-    cleanupV2IAMObjectsByIDPrefixes(idToken: string, idPrefix: string, objectPlurals: string[]): void
-    cleanupUsersByNamePrefix(idToken: string, namePrefix: string): void
-    cleanupTeamsByDescriptionPrefix(idToken: string, namePrefix: string): void
+    cleanupV2IAMObjectsByIDPrefixes(idPrefix: string, objectPlurals: string[]): void
```

Comment:
as an added bonus to this change, I dropped the idToken from all of these helpers. Before, we had to pass the admin user's bearer token for all of these API requests. Now we can just use the admin API token available as an environment variable.